Description: pacrunner: mozjs: mozjs185 is not always properly versioned.
 We have to assume that Mozilla knows what they are doing and thus, if we
 find mozjs185.pc, we accept it as being present.

--- a/libproxy/cmake/modules/pacrunner_mozjs.cmk
+++ b/libproxy/cmake/modules/pacrunner_mozjs.cmk
@@ -9,7 +9,7 @@ if(WIN32)
 elseif(NOT APPLE)
   option(WITH_MOZJS "Search for MOZJS package" ON)
   if (WITH_MOZJS)
-    pkg_search_module(MOZJS mozjs185>=1.8.5)
+    pkg_search_module(MOZJS mozjs185)
     if(MOZJS_FOUND)
       include_directories(${MOZJS_INCLUDE_DIRS})
       link_directories(${MOZJS_LIBRARY_DIRS})
